engine/table/src/main/java/io/deephaven/db/tables/utils/kosak/ParseDenseStorageToColumn.java

```diff
@@ -0,0 +1,115 @@
+package io.deephaven.db.tables.utils.kosak;
+
+import io.deephaven.db.tables.utils.kosak.context.NullContext;
+import io.deephaven.db.tables.utils.kosak.context.ParseContext;
+import io.deephaven.db.tables.utils.kosak.densestorage.DenseStorageReader;
+import io.deephaven.db.tables.utils.kosak.parsers.*;
+import io.deephaven.db.tables.utils.kosak.sinks.Sink;
+import io.deephaven.db.tables.utils.kosak.tokenizers.Tokenizer;
+import io.deephaven.db.tables.utils.kosak.results.ColumnBolus;
+import io.deephaven.db.tables.utils.kosak.util.Renderer;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public final class ParseDenseStorageToColumn {
```

Comment:
This comment is just an experiment:  Since Corey is planning to rename the package and move this file, I want to see what happens with a github review comment when a file moves; whether github is smart enough to keep the comment etc.

Reply:
Looks like the comment is still here?

Reply:
Yep! github is smart.